src/libraries/System.IO.Ports/tests/SerialPort/BaudRate.cs

```diff
@@ -316,7 +315,7 @@ private void VerifyBaudRate(SerialPort com1)
                 if (MAX_ACCEPTABLE_PERCENTAGE_DIFFERENCE < percentageDifference)
                 {
                     Assert.True(false, string.Format("ERROR!!! BuadRate not used Expected time:{0}, actual time:{1} percentageDifference:{2}",
```

Comment:
As an aside, not your concern for this PR, but these tests are old and don't meet the style guidelines of this repo. Eg., MAX_ACCEPTABLE_PERCENTAGE_DIFFERENCE  should be MaxAcceptablePercentageDifference.